examples/example_gene_fusion_class.json

```diff
@@ -1,103 +1,84 @@
-            "name": "tyrosine kinase catalytic domain",
-            "id": "interpro:IPR020635",
+            "status": "lost",
+            "name": "cystatin domain",
+            "id": "interpro:IPR000010",
             "gene": {
-                "value_id": "hgnc:8031",
-                "label": "NTRK1",
-                "type": "GeneDescriptor"
+                "id": "gene:CST1",
+                "value_id": "hgnc:2743",
+                "label": "CST1"
             }
         }
     ],
-    "causative_event": {
-        "event_type": "rearrangement"
-    },
     "transcript_components": [
         {
-            "region": {
-                "seq_id": "ncbi:NC_000001.11",
-                "start": 1000,
-                "end": 5000,
-                "type": "SequenceLocation"
-            },
-            "component_type": "unknown_gene"
-        },
-        {
-            "region": null,
-            "component_type": "unknown_gene"
+            "component_type": "transcript_segment",
+            "transcript": "NM_152263.3",
+            "exon_start": 1,
+            "exon_start_offset": 0,
+            "exon_end": 8,
+            "exon_end_offset": 0,
+            "gene": "TPM3",
+            "component_genomic_region": {
+                "type": "LocationDescription",
+               "value": {
+                    "sequence_id": "",
+                    "type": "SequenceLocation",
+                    "interval": {
+                        "start": 154192135,
+                        "end": 154170399,
+                        "type": "SimpleInterval"
+                    }
+                }
+            }
         },
         {
-            "region": {
-                "chr": "12",
-                "start": "p12.1",
-                "end": "p12.16",
-                "type": "ChromosomeLocation"
-            },
-            "component_type": "unknown_gene"
+            "component_type": "gene",
+            "gene": {
+                "id": "gene:ALK",
+                "value_id": "hgnc:427",
+                "label": "ALK"
+            }
         },
         {
-            "transcript": "NM:152263.3",
-            "exon_start": 1,
-            "exon_start_offset": -9,
-            "exon_end": 8,
-            "exon_end_offset": 7,
-            "gene": {
-                "value_id": "hgnc:12012",
-                "label": "TPM3",
-                "type": "GeneDescriptor"
-            },
-            "component_genomic_region": {
-                "description": null,
+            "component_type": "linker_sequence",
+            "linker_sequence": {
+                "id": "sequence:ACGT",
+                "type": "SequenceDescriptor",
                 "value": {
-                    "seq_id": "ncbi:NC_000001.11",
-                    "start": 154170399,
-                    "end": 1556346346,
-                    "type": "SequenceLocation"
+                    "sequence": "ACGT",
+                    "type": "SequenceState",
```

Comment:
We should make an issue to use VRS 1.2 classes. `SequenceState` and `SimpleInterval` are deprecated.